assets/docs/neso613/ASR_TFLite/pre_trained_models/1.md

```diff
@@ -0,0 +1,9 @@
+# Placeholder neso613/ASR_TFLite/pre_trained_models/1
+Conformer: Convolution-augmented Transformer for Speech Recognition trained on [LibriSpeech](http://www.openslr.org/12) dataset.
+
+<!-- dataset: librispeech -->
+<!-- task: audio-transcription -->
```

Comment:
Would you say that `audio-transcription` is the same as Speech-to-Text? If so, could you use the `audio-stt` value instead? This would allow users to find your model more quickly if they search for Speech-to-Text models. Otherwise, you'd have to add a new entry into tags/task.yaml like
```
  - id: audio-transcription
    display_name: Audio Transcription
```

Reply:
ok. Thanks for suggestion. I'll update it accordingly.